public/scripts/data-util.js

```diff
@@ -112,13 +113,16 @@ var DataUtil = (function() {
     return step['counters'][group][counter] || defaultValue;
   }
 
-    function extractConfigurationProperties(step) {
-	if (step['configuration_properties'] === undefined) {
-	    return {};
-	}
-
-	return step['configuration_properties'];
+  function extractConfigurationProperties(step) {
+    if (step['configuration_properties'] === undefined) {
+      return {};
     }
+    return step['configuration_properties'];
+  }
+
+  function splitArgs(str) {
```

Comment:
Is this still necessary? It looks unused and above the args are being manually split